graph/SchemaGraph.java

```diff
@@ -491,11 +492,22 @@ private Statistics() {
             attTypesWithValueType = new ConcurrentHashMap<>();
         }
 
-        public long abstractTypeCount() {
+        public long abstractThingTypeCount() {
             Supplier<Integer> fn = () -> toIntExact(thingTypes().stream().filter(TypeVertex::isAbstract).count());
             if (isReadOnly) {
-                if (abstractTypeCount == UNSET_COUNT) abstractTypeCount = fn.get();
-                return abstractTypeCount;
+                if (abstractThingTypeCount == UNSET_COUNT) abstractThingTypeCount = fn.get();
+                return abstractThingTypeCount;
+            } else {
+                return fn.get();
+            }
+        }
```

Comment:
I see there was an inconsistency between the method name `abstractTypeCount()` and the fact that it gets the stream from `thingTypes()`. However, should the fix be in the method name or in the stream? 🤔 

If I look at the user of this method, I think the method name seems correct, and looks like the stream should have been changed?

Reply:
Good question

Since it was the implementation rather than the name that was wrong, I will fix the implementation.